test/test.js

```diff
@@ -13,6 +13,20 @@ describe("TeamCity build number", function() {
 		});
 	});
 
+	describe("pull request regex", function() {
+		it("should match pull request naming convention", function() {
+			var pull = "pull/33";
+			
+			assert.equal(pull, pull.match(index.PullRequestRegex)[0]);
+		});
+
+		it("should match merge pull request naming convention", function() {
+			var merge = "33/merge";
+			
+			assert.equal(merge, merge.match(index.PullRequestRegex)[0]);
```

Comment:
Shouldn't this be:
```suggestion
			assert.equal(33, pull.match(index.PullRequestRegex)[2]);
```

(I might have got those the wrong way round)